jmxtrans-core/src/test/java/com/googlecode/jmxtrans/cluster/ClusterConnectTest.java

```diff
@@ -0,0 +1,197 @@
+package com.googlecode.jmxtrans.cluster;
+
+import com.kaching.platform.testing.AllowDNSResolution;
+import com.kaching.platform.testing.AllowNetworkAccess;
+import com.kaching.platform.testing.AllowNetworkListen;
+import org.apache.commons.configuration.Configuration;
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.CuratorFrameworkFactory;
+import org.apache.curator.framework.imps.CuratorFrameworkState;
+import org.apache.curator.framework.recipes.locks.InterProcessMutex;
+import org.apache.curator.retry.ExponentialBackoffRetry;
+import org.apache.zookeeper.CreateMode;
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Ignore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * ClusterConfigClient Tester.
+ *
+ * @author Tibor Kulcsar
+ * @version 1.0
+ * @since <pre>May 5, 2016</pre>
+ */
+@AllowDNSResolution
+@AllowNetworkListen(ports = {0})
+@AllowNetworkAccess(endpoints = {"127.0.0.1:*"})
+public class ClusterConnectTest {
+
+    CuratorFramework clClient;
+    Configuration clConfig;
+
+    private static final Logger log = LoggerFactory.getLogger(ClusterConnectTest.class);
+
+    @Before
+    public void before() throws Exception {
```

Comment:
Looks like you could use `@BeforeClass` instead of `@Before` and remove the conditional logic.